src/libs/test/TransactionRecoveryAPI/AxelarGMPRecoveryAPI.spec.ts

```diff
@@ -1139,6 +1143,60 @@ describe("AxelarGMPRecoveryAPI", () => {
     });
   });
 
+  describe("addGasToSuiChain", () => {
+    const network = "testnet";
+    // The default rpc url for testnet doesn't work as of 07 November 2024, so we need to use a custom one for testing.
+    const api: AxelarGMPRecoveryAPI = new AxelarGMPRecoveryAPI({ environment: Environment.DEVNET });
+    const testRpcUrl = "https://sui-testnet-rpc.publicnode.com";
```

Comment:
@pjenviri i used to get the 502 bad gateway error as well. i've tried using this rpc endpoint and it works, could you please try?

Reply:
Thanks, Euro. I'll try switching it in a bit.

Reply:
Ok, Sirapop did it a few mins ago, and it works! https://axelarscan-devnet-amplifier-git-feat-addgastosui-axelar-network.vercel.app/gmp/9UjHnbeQra9LBNLPHnmWZVR36wyUTf7qRd7qkaAMf4Ev-1